installation/developer-installation/octobot-developer-installation.md

```diff
@@ -207,3 +206,121 @@ this script will install the OctoBot-Tentacles folder and run Octobot with all y
 * **Script to zip, install and run in one go**
 
 ![](<../../.gitbook/assets/image (12).png>)
+
+
+## Setting up Visual Studio Code IDE
+
+### Open project folder in VS Code
+
+Open the OctoBot Project root folder with VS Code
+
+### Create Virtual Environment
+
+{% hint style="info" %}
+depending your system instead of "python" you might need to use "python3.8" or "python38" . Especially if you have multiple versions of python installed it's important to use the right version
+{% endhint %}
+
+```bash
+python -m venv venv  
+```
+
+### Activate virtual environment
+
+```bash
+.\venv\Scripts\activate
+```
+
+### Create VSCode launch.json
+
+![](../../.gitbook/assets/image.png)
+
+{% hint style="info" %}
+The VS Code launch config is locate in project-folder/.vscode/launch.json
```

Comment:
should this be added to OctoBot ? I see you closed the PR

Reply:
We could add it, but it would be on the wrong place anyway. Needs to be in the root of the project folder. So I decided to not add it there.
But maybe add it to a new project folder repo where vscode and pycharm scripts and settings are